shippable.resources.yml

```diff
@@ -3,7 +3,7 @@ resources:
 
   - name: vmk_app_img
     type: image
-    integration: drship_gcr
+    integration: drship_gcr # You need to replace this with your Google Cloud integration name
```

Comment:
we need to change the integration name here to the new google cloud integration. Not sure what the name should be so havent' changed till yet